controllers/users.js

```diff
@@ -24,6 +24,40 @@ const {
   USERS_PATCH_HANDLER_SUCCESS_MESSAGES,
 } = require("../constants/users");
 
+/**
+ * Patch Update user nickname
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const updateDiscordNicknames = async (req, res) => {
```

Comment:
discordId and username properties are both declared as const, which means that their values cannot be changed. However, the setUserDiscordNickname function is trying to change the value of the discordId property.

Reply:
In the **setUserDiscordNickname** function, I am passing the **discordId** through the request body
   `body: JSON.stringify({ userName, discordId })` but not changing it.